src/invite/migrations/0008_alter_noteinvitation_invite_type_and_more.py

```diff
@@ -0,0 +1,51 @@
+# Generated by Django 4.2.15 on 2024-09-26 23:33
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
What are the implications of adding these migrations? s this related to the notebook feature?
How does this change relate to new editor types?